core/services/relay/evm/codec.go

```diff
@@ -2,15 +2,24 @@ package evm
 
 import (
 	"encoding/json"
+	"fmt"
+	"math/big"
+	"reflect"
 
 	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/codec"
 
 	commontypes "github.com/smartcontractkit/chainlink-common/pkg/types"
 
 	"github.com/smartcontractkit/chainlink/v2/core/services/relay/evm/types"
 )
 
-func NewCodec(conf types.CodecConfig) (commontypes.Codec, error) {
+var evmDecoderHooks = []mapstructure.DecodeHookFunc{decodeAccountHook, codec.BigIntHook, codec.SliceToArrayVerifySizeHook, sizeVerifyBigIntHook}
```

Comment:
Decoder hooks are for evm types that are trickier to decode? Maybe just add a small comment above this that says something like that so that its easy to understand what these hooks are for at a glance